target_stitch/__init__.py

```diff
@@ -608,11 +608,11 @@ def handle_line(self, line):
                 self.flush()
 
             self.messages[current_stream].append(message)
-            self.buffer_size_bytes += len(line)
+            self.buffer_size_bytes[current_stream] = self.buffer_size_bytes.get(current_stream, 0) + len(line)
             if isinstance(message, singer.ActivateVersionMessage):
                 self.contains_activate_version[current_stream] = True
 
-            num_bytes = self.buffer_size_bytes
+            num_bytes = sum(self.buffer_size_bytes.values())
```

Comment:
Does a test cover this case? (It should, but want to double check)

Reply:
Good that it's still flushing based on total bytes

Reply:
Just added a test for this.